pkg/apis/componentconfig/cloudcore/v1alpha1/validation/validation.go

```diff
@@ -43,6 +44,25 @@ func ValidateCloudCoreConfiguration(c *v1alpha1.CloudCoreConfig) field.ErrorList
 	return allErrs
 }
 
+func ValidateCommonConfig(c v1alpha1.CommonConfig) field.ErrorList {
+	allErrs := field.ErrorList{}
+	validPort := utilvalidation.IsValidPortNum(int(c.MetricServer.Port))
+	validAddress := utilvalidation.IsValidIP(c.MetricServer.Address)
+
+	if len(validPort) > 0 {
```

Comment:
no need to check length? And how about adding CommonConfig field `TunnelPort` validation too?

Reply:
oh, tunnelPort needs being negotiated later.

Reply:
> no need to check length? And how about adding CommonConfig field `TunnelPort` validation too?

The variable names are confusing, rename to validPortErr